explainers/navigation-to-unsigned-bundles.md

```diff
@@ -375,6 +375,38 @@ full URLs needed by web bundles.
   * Opera: ?
   * Edge: ?
 * Web developers: ?
+* [ESCAPE workshop](https://tools.ietf.org/html/draft-iab-escape-report-00)
+  attendees:
+  * "Several participants reported that in areas where Internet access is
+    expensive, slow, or intermittent, the use of direct peer-to-peer file
+    exchange (e.g., "saving a Web site and sharing it on a USB stick") is
+    commonplace.  Most Web browsers already have some affordances for this, but
+    these are recognized as in need of improvements. … Many participants pointed
+    out that using "unsigned bundles" - that is, Web Packages without Signed
+    Exchanges - could be adequate for this use case, since most users don't need
+    cryptographic proof of the site's identity.  However, some expressed
+    concerns that this might worsen the propagation of falsehood." — [section
+    2.2](https://tools.ietf.org/html/draft-iab-escape-report-00#section-2.2)
+  * [Dave Cramer of
+    Hachette](https://www.iab.org/wp-content/IAB-uploads/2019/06/cramer-position-paper.pdf)
+    expressed "hope for a packaging model", but was concerned by several
+    questions: "How would packaging work with paid content, from paywalls to
+    micropayments to the traditional ebook model of selling content directly?
+    This is our big question. Advertising shouldn’t be the only model for
+    supporting content creation on the web, and the web shouldn’t be designed
+    around only this one model. We have many other questions, of course. How
+    would unsigned exchanges work? What happens when you GET a package? Can
+    packages be used for ad-hoc content distribution? I email you a package; you
+    double-click and it opens in your browser. How can web packaging become as
+    simple as PDF?"
+  * ["From an archival standpoint, the potential for Web content to be provided
+    in a self-contained form was viewed
+    positively."](https://tools.ietf.org/html/draft-iab-escape-report-00#section-2.3.2),
+    but archivists were worried that content producers might not package their
+    content enough to make it fully effective. They also thought packages might
+    improve the "situation for archival replay".
+* Computer vendors: Interest in using bundles to represent pre-installed
```

Comment:
* [Dan York of Internet Society](https://www.iab.org/wp-content/IAB-uploads/2019/06/dan-york.pdf) offered use cases for packaging web content focused on availability in low bandwidth / high latency environments. More specifically, part of the ISOC's mission involves reaching people who only have have limited connectivity to the Internet, e.g. WiFi at a few local places that people visit to download content that they want to read or watch later. The ability to package web content would be useful in such context. The ISOC is also working with communities where the electricity is unreliable, which negatively impacts Internet access as a result. Here again, these communities could benefit from the ability to download a package of content, either for learning or reading even in the absence of connectivity.

Reply:
Sorry for the delay, I think this captures what Dan wrote in his position paper but would welcome any feedback or direct edits.

Reply:
Thanks for the text. After a long delay, it's merged. Everything look good now?

Reply:
@jyasskin and @KenjiBaheux - Thanks for the summary and my apologies for a delay in replying. The text looks good overall, but I'd suggest two changes:

- we like to spell out "Internet Society" versus using "ISOC" (two many acronym conflicts in the larger world)
- I would add in a mention of satellite connectivity.

Perhaps the text would be better as:

Dan York of Internet Society offered use cases for packaging web content focused on availability in low bandwidth / high latency environments. More specifically, part of the Internet Society's work involves reaching people who only have have limited connectivity to the Internet, e.g. WiFi at a few local places that people visit to download content that they want to read or watch later. The ability to package web content would be useful in such context. The Internet Society is also working with communities where the electricity is unreliable, which negatively impacts Internet access as a result. There are also cases where communities only have periodic connectivity via satellite connections. Here again, these communities could benefit from the ability to download a package of content, either for learning or reading even in the absence of connectivity.

Reply:
Thanks! I've taken that wording.